include/hermes/Support/BigIntSupport.h

```diff
@@ -107,7 +107,8 @@ inline unsigned constexpr maxCharsPerDigitInRadix(uint8_t radix) {
   // With minNumBitsPerChar being the lower bound estimate of how many bits each
   // char can represent, the upper bound of how many chars "fit" in a bigint
   // digit is ceil(sizeofInBits(bigint digit) / minNumBitsPerChar).
-  unsigned numCharsPerDigits = BigIntDigitSizeInBits / (1 << minNumBitsPerChar);
+  unsigned numCharsPerDigits =
+      static_cast<unsigned>(BigIntDigitSizeInBits) / (1 << minNumBitsPerChar);
```

Comment:
Instead of casting, can you make BigIntDisitSizeInBits a uint32_t? It doesn't need to be 64-bits.

Reply:
I have changed the constants to `unit32_t`.

Reply:
I have also updated the comments.